tempest/api/workloadmgr/barbican/test_barbican.py

```diff
@@ -488,7 +495,6 @@ def test_01_barbican(self):
                     LOG.debug("***MDSUMS MATCH***")
                     reporting.add_test_step(
                         "Md5 Verification for boot disk", tvaultconf.PASS)
-                    tests[6][1] = 1
```

Comment:
Any specific reason for removing this assignment ?

Reply:
we are updating the var in both "pass and fail" case, it is added at line no.507, hence removed as redundant here.